### PR TITLE
Fix cp2kbindingenergytest

### PR DESCRIPTION
**Pull request by mpougin**
#85 
fixes as discussed in issue

### CODE REVIEW

File: examples/test_Cp2kBindingEnergy_CO2_MOF74.py [hunk 1]
```diff
@@ -63,7 +67,7 @@ def run_binding_energy_co2_mof74(cp2k_code, zn_mof74, co2_in_mof74):  # pylint:
         'num_mpiprocs_per_machine': 1,  # increase this to 4 in order to speed up the calculation
     }
     builder.cp2k_base.cp2k.metadata.options.withmpi = False  # comment this for parallel cp2k executable
-    builder.cp2k_base.cp2k.metadata.options.max_wallclock_seconds = 1 * 5 * 60
```
Comment (ltalirz) [inline examples/test_Cp2kBindingEnergy_CO2_MOF74.py:66]: are 5 minutes too short here?

**Review by ltalirz — COMMENTED** [review]
thanks @mpougin -  just one question here

File: examples/test_Cp2kBindingEnergy_CO2_MOF74.py (hunk 1 above)
Comment (mpougin) [reply examples/test_Cp2kBindingEnergy_CO2_MOF74.py:66]: yes, it ran out of walltime. It took ~30 minutes, so I just increased to 1h
Unbenannt6

**Comment by ltalirz** [comment]
thanks, just a minor pre-commit nitpick
https://github.com/lsmo-epfl/aiida-lsmo/pull/86/checks?check_run_id=2611058143#step:5:37

you'll see those locally if you do 
```
pip install -e .[pre-commit]
pre-commit install
```

**Comment by mpougin** [comment]
> thanks, just a minor pre-commit nitpick
> https://github.com/lsmo-epfl/aiida-lsmo/pull/86/checks?check_run_id=2611058143#step:5:37
> 
> you'll see those locally if you do
> 
> ```
> pip install -e .[pre-commit]
> pre-commit install
> ```

thanks for your remarks @ltalirz. stupid question, not sure if I understand properly: the pre-commit test failed because I didn't have pre-commit installed, which looks for minor issues before submission. Is this correct? I installed it now, do I have to redo everything?

**Comment by ltalirz** [comment]
> thanks for your remarks @ltalirz. stupid question, not sure if I understand properly: the pre-commit test failed because I didn't have pre-commit installed, which looks for minor issues before submission. Is this correct?

Correct.

> I installed it now, do I have to redo everything?

By default, pre-commit only scans the files that have uncommitted changes - therefore if you don't touch the file with the reported issue it won't complain (one can always run it on the whole codebase with `pre-commit run --all-files` but that can take a bit of time).

I.e. simply make the fix that was suggested and commit - if the pre-commit checks pass locally, so should they on github

**Comment by ltalirz** [comment]
thanks @mpougin !

**Pull request merged**